cmd/osv-scanner/internal/helper/getters.go

```diff
@@ -48,13 +48,10 @@ func GetCommonScannerActions(cmd *cli.Command, scanLicensesAllowlist []string) o
 
 func GetExperimentalScannerActions(cmd *cli.Command) osvscanner.ExperimentalScannerActions {
 	return osvscanner.ExperimentalScannerActions{
-		Extractors: ResolveEnabledExtractors(
-			cmd.StringSlice("experimental-extractors"),
-			cmd.StringSlice("experimental-disable-extractors"),
-		),
-		Detectors: ResolveEnabledDetectors(
-			cmd.StringSlice("experimental-detectors"),
-			cmd.StringSlice("experimental-disable-detectors"),
-		),
+		ExtractorsEnabled:  cmd.StringSlice("experimental-extractors"),
```

Comment:
I think eventually we want to think about merging these into a single enabled/disabled plugins list to match the scalibr behavior, but happy to leave it as it is for now until we get enrichers working.

Reply:
yeah that's what I was planning to do next and I've actually started on that locally - I think I'm about halfway through but then I realized having the extractors and detectors using slightly different structures (string vs `InitMap`) made it harder, so then I looked into https://github.com/google/osv-scanner/pull/2103 which looks mostly working